plugins/source/aws/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute to AWS source Plugin
+
+Welcome to CloudQuery contribution guide for AWS Source plugin!
+
+## Links
+
+- Homepage: [https://cloudquery.io](https://cloudquery.io)
+- Documentation: [https://docs.cloudquery.io](https://docs.cloudquery.io)
+- Tables: [./docs/tables/README.md](./docs/tables/README.md)
+- Discord: [https://cloudquery.io/discord](https://cloudquery.io/discord)
+- [Open AWS Issues](https://github.com/cloudquery/cloudquery/issues?q=is%3Aopen+is%3Aissue+label%3Aaws)
```

Comment:
```suggestion
- [Homepage](https://www.cloudquery.io)
- [Documentation](https://www.cloudquery.io/docs)
- [Tables](./docs/tables/README.md)
- [Discord](https://www.cloudquery.io/discord)
- [Open AWS Issues](https://github.com/cloudquery/cloudquery/issues?q=is%3Aopen+is%3Aissue+label%3Aaws)
```

Not sure where we landed with tables docs, but that link is broken at the moment